modules/mono/glue/Managed/Files/Mathf.cs

```diff
@@ -261,6 +261,11 @@ public static real_t Sinh(real_t s)
             return (real_t)Math.Sinh(s);
         }
 
+        public static real_t SmoothStep(real_t from, real_t to, real_t weight) {
```

Comment:
The opening brace should go into a new line.

Reply:
Fixed